tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -200,34 +244,64 @@ class MklConvFwdPrimitive : public MklPrimitive {
           bias_md(nullptr),
           fwd_pd(nullptr),
           conv_fwd(nullptr),
-          fwd_stream(nullptr) {}
+          fwd_stream(nullptr) {
+    }
   };
 
   void Setup(const MklConvFwdParams& convFwdDims) {
     // Create memory descriptors for convolution data w/ no specified format
     context_.src_md.reset(new memory::desc(
+#ifdef ENABLE_MKLDNN_V1
+        {convFwdDims.src_dims}, MklDnnType<Tinput>(), memory::format_tag::any));
```

Comment:
You can define this at the beginning of file
```
#ifdef ENABLE_MKLDNN_V1
#define MEMORY_FORMAT_TAG memory::format_tag
#else 
#define MEMORY_FORMAT_TAG memory::format
#endif
```
(Please also `#undef` at the end of file.)
Or you can use `using` instead of `#define`.

Same goes for other applicable places.

Reply:
Done.